README.md

```diff
@@ -1,5 +1,7 @@
-# OpenMATERIAL
-A new ASAM standard with the target to develop a 3D model exchange format with physical material properties for virtual development, test and validation of automated driving.
+# ASAM OpenMATERIAL 3D
+ASAM OpenMATERIAL 3D is a new ASAM standard with the target to develop a 3D model exchange format with physical material properties for virtual development, test and validation of automated driving.
+
+The project group discussions led to renaming the standard from "ASAM OpenMATERIAL" to "ASAM OpenMATERIAL 3D".
```

Comment:
Maybe add a half sentence ", to emphasize that the standard not only covers material properties but also 3D geometry."

Reply:
Sentence added.